lib/solidus_importer/processors/variant.rb

```diff
@@ -33,6 +33,10 @@ def save_variant
         variant = prepare_variant
         prepare_context(entity: variant, new_record: variant.new_record?, success: variant.save)
       end
+
+      def variant?
+        @variant ||= (@data && @data['Variant SKU'].present?)
```

Comment:
Why checking for the presence of `@data` here and in other place? 
I thought we expected it to be there, is there a scenario in which it can end up being nil or blank?

Reply:
Because `@data.blank?` check is made after the line `@product = context.fetch(:product) if variant?` - perhaps it looks a bit weird.

Reply:
Updated to remove data empty check from processors